src/wui/sound_options.cc

```diff
@@ -129,7 +130,17 @@ SoundOptions::SoundOptions(UI::Panel& parent, UI::SliderStyle style)
              style == UI::SliderStyle::kFsMenu ? UI::PanelStyle::kFsMenu : UI::PanelStyle::kWui,
              0,
              0,
-             UI::Box::Vertical) {
+             UI::Box::Vertical),
+     custom_songset_(
+        this,
+        UI::PanelStyle::kFsMenu,
+        {0, 0},
+        _("Play your own music in-game"),
+        _(richtext_escape(
+           "You can play custom in-game music by placing your own music files in "
+           "‘<Widelands Home Directory>/music/custom_XX.ogg’ (where ‘XX’ are sequential "
+           "two-digit numbers starting with 00)")),
```

Comment:
There's a subtle bug here. xgettext will not pick up the translatable string because the "keyword" `_` is not directly prefixed to the string literal. And at runtime, Widelands would first replace the `<` with `&lt;` and then look for the msgid with the `&lt;` but the PO file would contain an msgid with the `<` character. So the string would always be displayed in English for 2 independent reasons.
```suggestion
        richtext_escape(
           _("You can play custom in-game music by placing your own music files in "
           "‘<Widelands Home Directory>/music/custom_XX.ogg’ (where ‘XX’ are sequential "
           "two-digit numbers starting with 00)")),
```